pressbooks/pressbooks-book-directory-fe
# Remove recommended filter and tag

This should fix #370  and #362 

This PR removes the Recommended facet filter as well as the Recommended tag.
It also improves the collection tags on the book card by not rendering an empty `ul` when there are no tags to be displayed.

I did not delete the `PbBooleanFilters.vue` so we can maybe use it in the future.

#### How to test

If running tests locally, make sure to update local indices with Steel's configuration changes.

* Facet filters should not have Recommended accordion option on the sidebar
* Books cards should not display Recommended tag anymore - only collection tags
* When inspecting book cards there should not be an empty `ul` when there are no collection tags
